src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIExtension.groovy

```diff
@@ -55,6 +57,8 @@ class WPIExtension {
 
             "smartDashboardVersion" : new Tuple("SmartDashboard", smartDashboardVersion, "smartdashboard"),
             "shuffleboardVersion" : new Tuple("Shuffleboard", shuffleboardVersion, "shuffleboard"),
+            "outlineViewerVersion" : new Tuple("OutlineViewer", smartDashboardVersion, "outlineviewer"),
+            "robotBuilderVersion" : new Tuple("RobotBuilder", shuffleboardVersion, "robotbuilder"),
```

Comment:
copy-paste error with `smartDashboardVersion` and `shuffleboardVersion`

Reply:
Fixed